pt-BR/mail/chrome/messenger/msgmdn.properties

```diff
@@ -38,17 +38,17 @@
 #                    see bugzilla bug 139615.
 ## Msg Mdn Report strings
 MsgMdnDisplayed=Nota: esta confirmação de leitura somente informa que a mensagem foi aberta no computador do destinatário. Não há garantia que o destinatário tenha lido ou compreendido o conteúdo da mensagem.
-MsgMdnDispatched=A mensagem foi impressa, enviada por fax ou encaminhada sem que fosse aberta pelo destinário. Não há garantia de que o destinatário lerá a mensagem posteriormente.
-MsgMdnProcessed=A mensagem foi processada pelo aplicativo de email do destinatário sem que fosse aberta. Não há garantia de que a mensagem será lida posteriormente.
-MsgMdnDeleted=A mensagem foi excluída. O destinário pode ou não pode tê-la lido. Ele poderia retirá-la da lixeira posteriomente e abrí-la.
+MsgMdnDispatched=A mensagem foi impressa, enviada por fax ou encaminhada sem que fosse aberta pelo destinatário. Não há garantia de que o destinatário lerá a mensagem posteriormente.
+MsgMdnProcessed=A mensagem foi processada pelo aplicativo de e-mail do destinatário sem que fosse aberta. Não há garantia de que a mensagem será lida posteriormente.
```

Comment:
CORRECTIONS NEEDED it should be MsgMdnProcessed=A mensagem foi processada pelo aplicativo de e-mail do destinatário sem que fosse aberta. Não há garantia de que a mensagem seja lida posteriormente.